README.md

```diff
@@ -0,0 +1,26 @@
+# HAIC Telemetry Export
+
+## User Guide
+
+### Prerequisites
+
+- Make sure you have following installed
+    - kubectl
+    - AWS CLI
```

Comment:
We need to execute it on-prem - no AWS.

Reply:
Sure @mmalohlava  will look into it. Here AWS CLI is needed only to get the access to the cluster (update the `kubeconfig`) once it is done kubectl is used.